### PR TITLE
Add persian translation

### PR DESCRIPTION
**Pull request by Enlightments**
This is the Persian translation of the project.
Please feel free to revert feedback if there are needs for changes.

"src/_locales/fa/messages.json" file is added and contains Persian translation.

### CODE REVIEW

File: src/_locales/fa/messages.json [hunk 1]
```diff
@@ -0,0 +1,431 @@
...
+    "message": "فعال کردن حالت عیب‌یابی",
+    "description": "This is an option shown in the add-on settings."
+  },
+  "optionDebugModeDescr": {    
+    "message": "این گزینه برای دریافت اطلاعات بیشتر از جنس لاگ میباشد(برای گزارش و پیدا کردن خطا استفاده می شود)",
+    "description": "The description shown for the debug mode (optionDebugMode)."
+  },
+
+  "translatorCredit": {
+    "message": "This add-on has been translated into English by $TRANSLATORS$.",
```
Comment (rugk) [inline src/_locales/fa/messages.json]: This message also has to be translated. And say "Persian" of course, not English.

See https://github.com/TinyWebEx/common/blob/master/CONTRIBUTING.md#translations for how these placeholders are translated, especially [the official doc](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/Internationalization#Providing_localized_strings_in__locales).

Comment (rugk) [reply src/_locales/fa/messages.json]: Ah, actually I see you've duplicated the line, that seems to be the error here.

File: src/_locales/fa/messages.json [hunk 2]
```diff
@@ -0,0 +1,431 @@
...
+  "subtitleMisc": {
+    "message": "تنظیمات متفرقه",
+    "description": "The heading/grouping title shown for the miscellaneous setting of the QR code."
+  },
+  "optionQrQuietZone": {
+    "message": "مقدار فضای بیرونی کد کیو ار",
+    "description": "This is an option shown in the add-on settings. It's about the area around the QR code."
+  },
+  "optionQrQuietZoneStatusSingular": {
+    "message": "($INTEGER$ square)",
```
Comment (rugk) [inline src/_locales/fa/messages.json]: not translated

File: src/_locales/fa/messages.json [hunk 3]
```diff
@@ -0,0 +1,431 @@
...
+    "description": "This is the number of empty/'white' modules around the QR code in its singular form.",
+    "placeholders": {
+      "integer": {
+        "content": "$1",
+        "example": "1"
+      }
+    }
+  },
+  "optionQrQuietZoneStatusPlural": {
+    "message": "($INTEGER$ squares)",
```
Comment (rugk) [inline src/_locales/fa/messages.json]: not translated

File: src/_locales/fa/messages.json [hunk 4]
```diff
@@ -0,0 +1,431 @@
...
+    "description": "This is the number of empty/'white' modules around the QR code in its plural form.",
+    "placeholders": {
+      "integer": {
+        "content": "$1",
+        "example": "3"
+      }
+    }
+  },
+  "optionQrQuietZoneDescr": {
+    "message": "The number of border modules around the QR code colored in the background color.",
```
Comment (rugk) [inline src/_locales/fa/messages.json]: not translated

**Comment by rugk** [comment]
Hi @Enlightments,
thanks for your first contribution to this project! :tada: :+1: 🏅
I hope you'll like this project and enjoy hacking on it… :smiley:

Actually, it would be great [if you could try out the development version](https://github.com/TinyWebEx/common/blob/master/CONTRIBUTING.md#coding) and report back whether you like it or what may still be adjusted, so I can do this before the next release. In case there is something to be done… :smile: 

Just so we know the translation works good in the browser when it is actually used.

---

Please subscribe to #138 to get notified for new updates on translations etc. :smiley:

Also notice there are more things to translate like the add-on listings on AMO (addons.mozilla.org) etc. [See the contributing guide for more information.](https://github.com/TinyWebEx/common/blob/master/CONTRIBUTING.md#translation-of-add-on-listings)

**Comment by Enlightments** [comment]
Hi  @rugk 

Thanks for your feedback :smiley: 

Actually, I'm a first-timer contributor to opensource projects so kindly accept my apologize for possible errors and miss-understanding.

I will update the changes ASAP.

**Comment by rugk** [comment]
Nothing to apologize for. That's what reviews are for. :smiley:

**Comment by rugk** [comment]
@ Enlightments If you need any pointers/help or have questions, feel free to ask.

**Comment by rugk** [comment]
@Enlightments Are you still interested in completing this? If not, that's no problem, just add a notice, so I know what the current status of this PR is.

**Comment by rugk** [comment]
@Enlightments Okay, given that the current translation is better than nothing, I'll merge it. But if you'll find the time, I'd kindly ask you to contribute the missing bits. Thanks. :smiley:

**Comment by rugk** [comment]
Merged in https://github.com/rugk/offline-qr-code/commit/bdd9d33428f94202454be54a889693101867dd6b, which GitHub did not detect :roll_eyes:

**Pull request closed without merging**